yellowbrick/features/pcoords.py

```diff
@@ -169,6 +181,15 @@ class ParallelCoordinates(DataVisualizer):
         Keyword arguments that are passed to the base class and may influence
         the visualization as defined in other Visualizers.
 
+    Attributes
+    --------
+
+    n_samples_ : int
+        number of samples included in the visualization object
+
+    rng_ : np.random.RandomState
```

Comment:
This is a complete nit, and I apologize for it, but since I made a big deal of `n_samples_` I feel like I should point out that this is not a learned parameter in fit, but a private attribute mirroring `random_state` and should probably be `_rng`. If I didn't have any other comments I would have just ignored this and perhaps made a quick change after the PR, but perhaps you could quickly update it?

Reply:
Done.  Removed reference to _rng from docstring.